accepted/0042-isolated-mode.md

```diff
@@ -0,0 +1,347 @@
+# Isolated mode
+
+## Summary
+
+This RFC is a proposal to add a new opt-in installation mode.
+
+This mode aims at preventing workspaces from over-sharing their dependencies. With this mode, two workspaces will share a dependency only if both have declared it in their package.json.
+
+This RFC aims at improving projects with workspaces, but the suggested implementation will also happen to bring value to projects not using workspaces. Because `isolated-mode` is valuable for any project, it will also be available for projects not using workspaces.
```

Comment:
This is bit repetitive.

While this RFC aims at improving projects with workspaces, `isolated-mode` could be valuable for any project, whether or not it uses workspaces.